app/src/main/java/in/testpress/testpress/repository/RegisterRepository.kt

```diff
@@ -0,0 +1,36 @@
+package `in`.testpress.testpress.repository
+
+import `in`.testpress.testpress.core.TestpressService
+import `in`.testpress.testpress.util.SafeAsyncTask
+import androidx.lifecycle.MutableLiveData
+
+open class RegisterRepository(val testpressService: TestpressService) {
+
+    var isRegisterSuccessful = MutableLiveData<Boolean>()
```

Comment:
Also let it not be Boolean. We want the UI to inform what error occurred. Please use the Resource datatype `in/testpress/network/Resource.kt` to provide appropriate error message in case of error.

Reply:
Add test cases for the error messages too.